curves/bn254/conversions.go

```diff
@@ -0,0 +1,79 @@
+package bn254
+
+import (
+	bn254 "github.com/consensys/gnark-crypto/ecc/bn254"
+	"github.com/consensys/gnark-crypto/ecc/bn254/fp"
+	icicle_bn254 "github.com/ingonyama-zk/icicle/wrappers/golang/curves/bn254"
+)
+
+func StripZ(p *icicle_bn254.Projective) *icicle_bn254.Affine {
+	return &icicle_bn254.Affine{
+		X: p.X,
+		Y: p.Y,
+	}
+}
+
+func BatchConvertFromG1Affine(elements []bn254.G1Affine) []icicle_bn254.Affine {
+	var newElements []icicle_bn254.Affine
+	for _, e := range elements {
+		var newElement icicle_bn254.Projective
+		FromG1AffineGnark(&e, &newElement)
+
+		newElements = append(newElements, *StripZ(&newElement))
```

Comment:
We should use `newElement.ProjectiveToAffine`. I think it is more future proof for cases where z != 1